etc/kayobe/kolla.yml

```diff
@@ -123,7 +123,7 @@ kolla_build_blocks:
   monasca_persister_footer: |
     # NOTE(brtknr): Implement separate db per tenancy (stein backport):
     # https://review.opendev.org/#/c/674484/
-    RUN pip install -U --no-deps git+https://github.com/stackhpc/monasca-persister.git@stackhpc/1.14.0.2
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/monasca-persister.git@stein-retention-fix
```

Comment:
Sure we don't want to release this?

Reply:
@brtknr is this fixed merged and tagged now?

Reply:
Still not merged upstream. There is a chance it could changed before being merged upstream. See:
https://github.com/stackhpc/monasca-persister/pull/6
https://github.com/stackhpc/monasca-persister/pull/7